app/models/lookup_key.rb

```diff
@@ -158,6 +158,11 @@ def path_elements
     end
   end
 
+  def overriden_on_host?(host)
+    return false if host.is_a?(Hostgroup)
+    lookup_values.find_by_match("fqdn=#{host.fqdn}").present?
```

Comment:
that would be the same as returning true :)

a lookup key's lookup values have lookup_key_id == id by definition.
what we're looking is "is it overridden by the host we're looking at", not "is there something overriding this somewhere"